src/components/Breadcrumbs.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from "react";
+import { NavLink } from "react-router-dom";
+import styles from "./Breadcrumbs.css";
+import gridStyles from "./../styles/Grid.css";
+
+
+class Breadcrumbs extends Component {
+  constructor(props) {
+    super(props);
+    this.uuidRegex = /^[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$/i;
+  }
+  computeBreadcrumb() {
+    const { pathname } = this.props.location;
+    const splitPathnames = pathname.slice().split("/");
+    return pathname === "/"
+      ? null
+      : splitPathnames.map((sp, i) => {
+        const to = `/${splitPathnames.slice(1, i + 1).join("/")}`;
```

Comment:
to could be called url or relative url.
// splice from 1 and not 0 to remove first empty string from array that was caused by splitting on "/"